src/gui/scaling_icon_engine_plugin.cpp

```diff
@@ -22,7 +22,7 @@
 
 #include <QIconEnginePlugin>
 
-#include "scaling_icon_engine.h"
+#include "gui/scaling_icon_engine.h"
```

Comment:
I guess this is proposed for some sort of consistency. But for `#include "foo"`, "typical implementations first search the directory where the current file resides". so the original form seems appropriate.

Reply:
Yes, since we always prepend the path for any header file which is not the one of the source file itself, I consider that change a matter of style and consistency.